### PR TITLE
Fix createSideNFTToken on NFTBridge

### PR DESCRIPTION
**Pull request by marcciosilva**
- Add missing data to NFT token creation through new bridge
- Adjust factory and token contracts
- Add unit tests to all affected contracts

### CODE REVIEW

File: bridge/test/NFTBridge_test.js
Comment (pmprete) [inline bridge/test/NFTBridge_test.js:21]: you can import the null address from utils https://github.com/rsksmart/tokenbridge/blob/master/bridge/test/utils.js#L164

File: bridge/test/NFTBridge_test.js
Comment (pmprete) [inline bridge/test/NFTBridge_test.js:135]: As all this values are inmutable, maybe it's better to convert this beforeEach to before

File: bridge/test/NFTBridge_test.js
Comment (pmprete) [inline bridge/test/NFTBridge_test.js:148]: good use of truffleAssert

File: bridge/test/SideNFTTokenFactory_test.js
Comment (pmprete) [inline bridge/test/SideNFTTokenFactory_test.js:62]: If you want you can remove the test for the parameters as this is how the frameworks works

File: bridge/test/SideNFTToken_test.js
Comment (pmprete) [inline bridge/test/SideNFTToken_test.js:32]: Also this tests should include add minting Tests and transfer test to a wallet, a contract that does not implement ERC165 interfaces, and one that does
We could copy paste some of the open zeppelin tests for this. If we consider it an over kill, we can skip this until we are working on the MVP and have the ok from the security team to skip this tests

**Review by pmprete — APPROVED** [review]
Non of my comments is a blocker, so feel free to merge this and address the comments later on

File: bridge/test/NFTBridge_test.js
Comment (marcciosilva) [reply bridge/test/NFTBridge_test.js:135]: Actually the token contract is created on a beforeEach call from before (I think it makes sense to recreate the contract for each test to avoid a dirty context, right?) - so if we keep that, since this depends on it, we'd have to keep it this way

Comment (pmprete) [reply bridge/test/NFTBridge_test.js:138]: you are right, didn't notice the contract was on a beforeEach

File: bridge/test/SideNFTToken_test.js
Comment (pmprete) [reply bridge/test/SideNFTToken_test.js:25]: Just talked with Marccio, we will leave this tests when we start using the transfer and minting when we tackle the claim task

**Pull request merged**
